datasets/ehealth_kd/ehealth_kd.py

```diff
@@ -0,0 +1,159 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@InProceedings{huggingface:dataset,
+title = {A great new dataset},
+authors={huggingface, Inc.
+},
+year={2020}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+This new dataset is designed to solve this great NLP task and is crafted with a lot of care.
+"""
+
+_HOMEPAGE = "https://knowledge-learning.github.io/ehealthkd-2020/"
+
+_LICENSE = ""
+
+_URL = "https://raw.githubusercontent.com/knowledge-learning/ehealthkd-2020/master/data/"
+_TRAIN_DIR = "training/"
+_DEV_DIR = "development/main/"
+_TEST_DIR = "testing/scenario1-main/"
+_TEXT_FILE = "scenario.txt"
+_ANNOTATIONS_FILE = "scenario.ann"
+
+
+class eHealthKD(datasets.GeneratorBasedBuilder):
```

Comment:
I believe that `CamelCase` is the convention for `datasets` so we should probably capitalise the first letter as well, i.e. `EHealthKD(...)`

Reply:
You're right, thanks. Done!